orttraining/orttraining/training_ops/cuda/activation/activations_grad_impl.cu

```diff
@@ -31,6 +31,14 @@ struct OP_ReluGrad : public CtxReluGrad {
   }
 };
 
+template <typename T>
+struct OP_SigmoidGrad : public CtxSigmoidGrad {
+  __device__ __inline__ T operator()(const T& dy, const T& x) const {
```

Comment:
the second input should be O(0), aka y. 

return dy * y * (1-y)

Reply:
Yes. Isn't y=sigmoid(x)?